### DISCUSSION TITLE
Verifying a publisher on the marketplace

### DISCUSSION THREAD

**Discussion by pokey** [microsoft/vscode-discussions#75] [Extension Development QnA] [answered]
I submitted my publisher account ([`pokey`](https://marketplace.visualstudio.com/publishers/pokey)) for verification (like [so](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#verify-a-publisher)) more than a month ago, and although it says the team will get back to me in 5 business days, I’ve had no response.  

I’m not sure if this is the right place to ask, but I’m not sure how best to proceed.  Thanks for your help!

**Comment by alefragnani** [comment]
I'm not sure this will help, but the VS Marketplace team has a separate repo as well (https://github.com/microsoft/vsmarketplace/issues). 

Or maybe pinging some of the PMs from the VS Code / VS Marketplace teams: @digitarald , @olguzzar , @prashantvc 

Hope this helps

**Comment by pokey** [comment, marked as answer]
Was able to get this one resolved by email
